### DISCUSSION TITLE
[Feature] Hide images with specific tags from the timeline

### DISCUSSION THREAD

**Discussion by Lauritz-Tieste** [immich-app/immich#12610] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think it could be interesting if you could hide images that you have tagged from the main timeline.

Here's an example: In some places under my images there are documents, screenshots or images that are not interesting for the overall timeline. I would like to tag these manually and then no longer have them in the main timeline, but they should still be accessible via the tag view.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Ixiruss** [comment]
Hello,

Not sure it fit exactly your use case, but generally it would be useful to have an option to hide every assets tagged in the search filter.
It would be great for tagging purpose.

One use case I have in mind : 
- Lots of tagged photos/videos already in the timeline. We add few photos/videos that got flooded in the timeline (date of photos are not recent). You can't easily tags them as they are mixed with all tagged assets.

Not sure if I need to create a new feature request.

In any case, hope that can helps in some way to your request.

**Comment by ayalavalva** [comment]
Yes, this is an excellent idea. More generally, it would be great to introduce the concept of timeline "views." This feature would allow users to include or exclude specific tags from the search or timeline and save the search results as a customized timeline "view."

A typical use case, which I believe many users would find valuable, is the ability to filter out photos taken or shared by others (e.g., from social networks or chat apps).

Additionally, it would be helpful to have an option to automatically tag external library paths with specific tags, enhancing organization and filtering capabilities.

**Reply by Lauritz-Tieste** [reply]
Since this PR was closed due to duplicate, can you create a new one based on this idea?

**Comment by bo0tzz** [comment]
#3477
